blog/_src/posts/2016-10-31-meaningful-distinctions.md

```diff
@@ -0,0 +1,85 @@
+    Title: Meaningful Distinctions
+    Date: 2016-10-31T17:20:33
+    Tags: history, constructions, by Ben Greenman
+
+> "Meaningful distinctions deserve to be maintained." -- Errett A. Bishop
+
+Likewise, memorable quotations deserve to be read in context.
+In this spirit, I am happy to present the above "basic principle" in its context: 
+[_Schizophrenia in contemporary mathematics_ (pdf)](/img/sicm.pdf)
+
+Read on for a brief summary.
+
+<!-- more -->
+***
+
+I first read the above quotation in
+[Michael Beeson's introduction](http://www.michaelbeeson.com/research/papers/BishopForeword.pdf)
+to the 2012 edition of Bishop's [_Foundations of Constructive Analysis_](https://www.amazon.com/Foundations-Constructive-Analysis-Errett-Bishop/dp/4871877140).
+That was two years ago.
+
+Last month, I tried to find its context.
+[Many](https://books.google.com/books?id=uPx8tGCaxzUC&pg=PA214&lpg=PA214&dq=meaningful+distinctions+deserve+to+be+maintained&source=bl&ots=cWjwOTnNuT&sig=wN143wNyfXtMFLGABBQM-22aSOQ&hl=en&sa=X&ved=0ahUKEwjPwt2HmYbQAhWE6IMKHU5rB8YQ6AEIHjAA#v=onepage&q=meaningful%20distinctions%20deserve%20to%20be%20maintained&f=false)
+[other](https://www.jstor.org/stable/2589553)
+[uses](https://books.google.com/books?id=J4DkBwAAQBAJ&pg=PA6&lpg=PA6&dq=meaningful+distinctions+deserve+to+be+maintained&source=bl&ots=KYkrkBrJd_&sig=AAK1A_uIkQlVcYCY1TFljfA3CqA&hl=en&sa=X&ved=0ahUKEwjPwt2HmYbQAhWE6IMKHU5rB8YQ6AEIJTAC#v=onepage&q=meaningful%20distinctions%20deserve%20to%20be%20maintained&f=false)
+[of](https://books.google.com/books?id=oN5nsPkXhhsC&pg=PR6&lpg=PR6&dq=meaningful+distinctions+deserve+to+be+maintained&source=bl&ots=4doTufVdsy&sig=u3e_Z_xdN-tjt9p1eqQ88juA5Ns&hl=en&sa=X&ved=0ahUKEwjPwt2HmYbQAhWE6IMKHU5rB8YQ6AEIKDAD#v=onepage&q=meaningful%20distinctions%20deserve%20to%20be%20maintained&f=false)
+[the](https://books.google.com/books?id=GR44SKXCZJsC&pg=RA1-PA199&lpg=RA1-PA199&dq=meaningful+distinctions+deserve+to+be+maintained&source=bl&ots=lNpzR5QV7h&sig=IGg2Q_KtreSAhrbSJxsV7mQ8xok&hl=en&sa=X&ved=0ahUKEwjPwt2HmYbQAhWE6IMKHU5rB8YQ6AEIMDAF#v=onepage&q=meaningful%20distinctions%20deserve%20to%20be%20maintained&f=false)
+[quote](http://www.math.canterbury.ac.nz/php/groups/cm/faq/)
+[cited](http://www.ben-sherman.net/aux/evident-logic.pdf)
+a
+_Schizophrenia in comtemporary mathematics_, but I could not find an electronic copy.
+(It turns out, the AMS Bookstore page for [_Erret Bishop: Reflections on Him and His Research_](http://bookstore.ams.org/conm-39) includes a facsimile.)
+
+Lest anyone else is tempted to conjure the ancient magic of inter-library loan,
```

Comment:
This is pretty archaic phrasing regardless, but shouldn't it be "Lest anyone else be tempted ..."?